packages/framework/src/Model/Product/Pricing/QuantifiedProductPriceCalculation.php

```diff
@@ -64,7 +66,7 @@ public function __construct(
      * @param \Shopsys\FrameworkBundle\Model\Customer\User|null $user
      * @return \Shopsys\FrameworkBundle\Model\Order\Item\QuantifiedItemPrice
      */
-    public function calculatePrice(QuantifiedProduct $quantifiedProduct, $domainId, User $user = null)
+    public function calculatePrice(QuantifiedProduct $quantifiedProduct, int $domainId, User $user = null): QuantifiedItemPrice
```

Comment:
This is the most WTF class I've seen so far in SSFW.... where is it even used? Where is it constructed? How... and why?

Reply:
It's used in `OrderPreviewCalculation`... What confuses you is probably the fact it uses properties instead of parameters for passing values in between methods. I'll try to fix that.

Reply:
See f4bc2a8c4, the methods make more sense and we removed unnecessary calls.